AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/ads/applovin/banners/BannerProgrammaticActivity.kt

```diff
@@ -33,15 +33,18 @@ class BannerProgrammaticActivity : BaseAdActivity(),
         adView.setAdClickListener(this)
 
         adView.id = ViewCompat.generateViewId()
-        load_button.setOnClickListener { adView.loadNextAd() }
+        val loadButton = findViewById<Button>(R.id.load_button)
+        loadButton.setOnClickListener { adView.loadNextAd() }
 
         // Add programmatically created banner into our container
-        banner_programmatic_content_layout.addView(adView, ConstraintLayout.LayoutParams(ViewGroup.LayoutParams.MATCH_PARENT, AppLovinSdkUtils.dpToPx(this, 50)))
+        val bannerProgrammaticContentLayout =
```

Comment:
Feel we don't need to be this verbose with the naming, e.g. `contentLayout` would be fine